dialects/datafusion.yaml

```diff
@@ -236,6 +236,12 @@ scalar_functions:
   - name: right
   - name: string_split
     unsupported: True
+  - name: starts_with
+    unsupported: True
+  - name: ends_with
+    unsupported: True
```

Comment:
Datafusion should have starts_with/ends_with support: 
https://datafusion.apache.org/python/generated/datafusion.functions.functions.html#datafusion.functions.functions.starts_with

Reply:
fixed it